### PR TITLE
Add com.baidu.BaiduNetdisk

### PR DESCRIPTION
**Pull request by cheese**
This flatpak is usable, but may not have reach the minimal acceptance. I will try to contact upstream to improve it.

Current issues:
- The appdata xml is primitive
  - No licence text or URL
  - Icon is too small
  - Screenshot is not available
- No language support other than Chinese

### CODE REVIEW

**Comment by cheese** [comment]
bot, build com.baidu.NetDisk

File: com.baidu.NetDisk.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          - rmdir opt
+          - mv usr export
+          - rm -fr export/share/doc
+          - desktop-file-edit --set-key="Exec" --set-value="baidunetdisk %U" export/share/applications/baidunetdisk.desktop
+          - desktop-file-edit --set-key="Icon" --set-value="com.baidu.NetDisk" export/share/applications/baidunetdisk.desktop
+          - desktop-file-edit --set-key="X-Flatpak-RenamedFrom" --set-value="baidunetdisk.desktop;" export/share/applications/baidunetdisk.desktop
+          - rename "baidunetdisk." "com.baidu.NetDisk." export/share/{icons/hicolor/*/*,applications}/baidunetdisk.*
+
+      - type: file
+        path: com.baidu.NetDisk.appdata.xml
```
Comment (LeandroStanger) [inline com.baidu.NetDisk.yml]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

**Comment by cheese** [comment]
bot, build com.baidu.BaiduNetdisk

**Comment by barthalion** [comment]
@cheese Feel free to ping me/us when you consider it ready. (I'd say it's fine to be Chinese-only as long as appdata says so.)

**Pull request closed without merging**